### DISCUSSION TITLE
Problem while using pgcron

### DISCUSSION THREAD

**Discussion by pichichi91** [supabase/supabase#2126] [Questions] [answered]
Hi 

I have been trying to follow the **[article](https://supabase.io/blog/2021/03/05/postgres-as-a-cron-server)** to add pgcron with http to my supabase database in order to try if I can have scheduled reports like that 😄 

I installed both plugins over the UI and scheduled a first job:

| jobid | schedule  | command                                                                                                                                                                                          | nodename  | nodeport | database | username       | active | jobname              |
| ----- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------- | -------- | -------- | -------------- | ------ | -------------------- |
| 3     | * * * * * |  select status from http_post( 'https://webhook.site/f6061c17-3f98-45f6-834f-e1e6106f756e',  '{"hello": "world"}', 'application/json') | localhost | 5432     | postgres | supabase_admin | true   | webhook-every-minute |

But somehow it's never called - Anyone has an idea how I can further troubleshoot this? (Calling the http_host manually has worked perfectly for me)

No rows returned for `select * from cron.job_run_details`

**Comment by inian** [comment, marked as answer]
Hi @pichichi91, sorry about this. This is a known issue https://github.com/supabase/supabase/issues/1703 and we are working on the fix

**Reply by pichichi91** [reply]
oh wow - this was an quick answer 😱 thanks for the reponse
